runtime/doc/message.txt

```diff
@@ -315,10 +315,9 @@ generate this message.  This could be caused by a |-m| commandline argument.
 You can switch the 'write' option on with ":set write".
 
 							*E25*  >
-  GUI cannot be used: Not enabled at compile time
+  Nvim does not have a built-in GUI
```

Comment:
Ok, I see that you have '*a* built-in GUI' everywhere except in the actual message in `globals.h`. So just add the article there and it should be fine.